### PR TITLE
Release followup

### PR DESCRIPTION
**Pull request by olafurpg**
Minor build changes after v0.6.6 release

### CODE REVIEW

File: bin/testRelease.sh [hunk 1]
```diff
@@ -2,17 +2,19 @@
...
+  com.lihaoyi:utest_native0.3_2.11:$VERSION \
+  com.lihaoyi:utest_sjs0.6_2.10:$VERSION \
+  com.lihaoyi:utest_sjs0.6_2.11:$VERSION \
+  com.lihaoyi:utest_sjs0.6_2.12:$VERSION \
+  com.lihaoyi:utest_sjs0.6_2.13.0-M5:$VERSION \
+  com.lihaoyi:utest_sjs0.6_2.13.0-M5:$VERSION \
+  com.lihaoyi:utest_sjs1.0.0-M5_2.11:$VERSION \
+  com.lihaoyi:utest_sjs1.0.0-M5_2.12:$VERSION \
+  -r sonatype:releases
+  # com.lihaoyi:utest_sjs1.0.0-M5_2.13.0-M5:$VERSION \ # Did not release
```
Comment (olafurpg) [inline bin/testRelease.sh]: @sjrd is this resolution error expected?

```
[error] sbt.librarymanagement.ResolveException: unresolved dependency: org.scala-js#scalajs-library_2.13.0-M5;1.0.0-M5: not found
```

Happened during release https://travis-ci.org/lihaoyi/utest/jobs/433990118#L2762

Comment (sjrd) [reply bin/testRelease.sh]: Yes. Scala.js 1.0.0-M5 does not support Scala 2.13.0-M5.

Comment (olafurpg) [reply bin/testRelease.sh]: OK, let me disable it in .travis.yml then.

**Comment by SethTisue** [comment]
LGTM

**Pull request merged**